### PR TITLE
Div 4138 use govuk frontend

### PR DESCRIPTION
**Pull request by dtaylor7**
[DIV-3893 - Update to GOV.UK Frontend](https://tools.hmcts.net/jira/browse/DIV-3893)
 - Remove govuk_template
 - Add govuk-frontend
 - Update html to work with new gov.uk frontend

### CODE REVIEW

File: steps/co-respondent/cr-choose-a-response/CrChooseAResponse.html [hunk 1]
```diff
@@ -15,22 +15,18 @@
...
-        <strong class="bold-small">
-            {{ content.action | safe }}
-        </strong>
         {% call formSection() %}
-            {{ selectionButtons(fields.response, content.title,
+            {{ selectionButtons(fields.response, content.action,
                 options = responseOptions,
                 hint=false,
-                hideQuestion=true,
+                hideQuestion=false,
```
Comment (yarpo) [inline steps/co-respondent/cr-choose-a-response/CrChooseAResponse.html]: are you sure this should be changed?

File: steps/respondent/progress-bar/ProgressBar.html [hunk 2]
```diff
@@ -1,48 +1,51 @@
...
 
 {% from "look-and-feel/components/document-list.njk" import documentList %}
+{% from "look-and-feel/components/warning-text.njk" import warningText %}
 
 {% block back %}
 {% endblock %}
 
-{% set title %}
-    {{ content.title | safe }}
+{% set title  %}
```
Comment (yarpo) [inline steps/respondent/progress-bar/ProgressBar.html]: probably this space is not needed ;) [ignore it, can be remove later, or not at all]

File: steps/respondent/progress-bar/ProgressBar.html [hunk 3]
```diff
@@ -1,48 +1,51 @@
...
+  {{ content.title | safe }}
 {% endset %}
 
 {% block main_content %}
-    <p class="font-xsmall">
-        <strong class="bold-xsmall">{{ content.details.referenceNumber }}
+    <p class="govuk-body-s">
+        <strong class="govuk-!-font-size-16">{{ content.details.referenceNumber }}
             :</strong>&nbsp;&nbsp;&nbsp; {{ session.originalPetition.caseReference }}<br>
```
Comment (yarpo) [inline steps/respondent/progress-bar/ProgressBar.html]: are we happy with `&nbsp;`s?

Comment (dtaylor7) [reply steps/respondent/progress-bar/ProgressBar.html]: not sure why they are there, but can be reviewed in tech debt?

File: steps/respondent/progress-bar/ProgressBar.html (hunk 2 above)
Comment (dtaylor7) [reply steps/respondent/progress-bar/ProgressBar.html]: yep will remove

File: package.json [hunk 4]
```diff
@@ -41,6 +41,7 @@
     "express": "^4.16.3",
     "express-limiter": "^1.6.1",
     "glob": "^7.1.2",
+    "govuk-frontend": "^2.11.0",
```
Comment (hosainnet) [inline package.json]: Should we actually be explicity adding it here? Thought i'd get bundled along with `look-and-feel` (like the previous one did)

File: steps/co-respondent/cr-choose-a-response/CrChooseAResponse.html (hunk 1 above)
Comment (dtaylor7) [reply steps/co-respondent/cr-choose-a-response/CrChooseAResponse.html]: yes, as i removed question a few lines above

File: package.json (hunk 4 above)
Comment (dtaylor7) [reply package.json]: yep will remove 👍

Comment (hosainnet) [reply package.json]: Thanks, will probably need to do the same for DN in a new PR https://github.com/hmcts/div-decree-nisi-frontend/pull/370/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R54

File: test/w3cjs/index.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 const w3cjs = require('w3cjs');
-const steps = require('steps')();
+const steps = require('steps')(false);
```
Comment (hosainnet) [inline test/w3cjs/index.js]: `false`?

File: test/unit/steps/corespondent/crReviewApplication.test.js [hunk 6]
```diff
@@ -98,7 +98,7 @@ describe(modulePath, () => {
         CrReviewApplication,
         session,
         {
-          specificValues: [ session.caseReference ]
+          specificValues: [ session.originalPetition.caseReference ]
```
Comment (hosainnet) [inline test/unit/steps/corespondent/crReviewApplication.test.js]: Hmm, how did this ever work?

Comment (dtaylor7) [reply test/unit/steps/corespondent/crReviewApplication.test.js]: null value

Comment (dtaylor7) [reply test/unit/steps/corespondent/crReviewApplication.test.js]: it was checking if nothing existed

File: test/w3cjs/index.js (hunk 5 above)
Comment (dtaylor7) [reply test/w3cjs/index.js]: https://github.com/hmcts/div-respondent-frontend/blob/master/steps/index.js#L4

Comment (dtaylor7) [reply test/w3cjs/index.js]: https://github.com/hmcts/div-respondent-frontend/blob/d626ecfde61ad04204ebe24c801b667d04cda3b3/steps/index.js#L4

**Pull request merged**